src/components/resource/resource-edit.js

```diff
@@ -141,16 +141,20 @@ export default class ResourceEdit extends Component {
           const { model } = this.props;
           const { customCoverages: customCoverageDateValues } = values;
           let coverageDates = model.managedCoverages;
-          if (customCoverageDateValues && customCoverageDateValues.length > 0) {
+
+          const customCoverageExists = customCoverageDateValues && customCoverageDateValues.length > 0;
+          if (customCoverageExists) {
```

Comment:
Empty line after declaration
```suggestion
          const customCoverageExists = customCoverageDateValues && customCoverageDateValues.length;
    
          if (customCoverageExists) {
```